samples/custom-checks/monitoring3rdparty/Version_6/3rdPartySystem/3rdPartySystem.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{6D5CB974-898B-4AD7-83FB-79C4477449FB}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>_3rdPartySystem</RootNamespace>
+    <AssemblyName>3rdPartySystem</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="app.manifest" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
```

Comment:
Ok I fix this on the v5 sample then as well

On Thursday, January 28, 2016, Simon Cropp <notifications@github.com> wrote:

> In
> samples/custom-checks/monitoring3rdparty/Version_6/3rdPartySystem/3rdPartySystem.csproj
> <https://github.com/Particular/docs.particular.net/pull/1104#discussion_r51108048>
> :
>
> > +    <Reference Include="System.Core" />
> > +    <Reference Include="System.Net.Http" />
> > +    <Reference Include="System.Xml.Linq" />
> > +    <Reference Include="System.Data.DataSetExtensions" />
> > +    <Reference Include="Microsoft.CSharp" />
> > +    <Reference Include="System.Data" />
> > +    <Reference Include="System.Xml" />
> > +  </ItemGroup>
> > +  <ItemGroup>
> > +    <Compile Include="Program.cs" />
> > +  </ItemGroup>
> > +  <ItemGroup>
> > +    <None Include="app.manifest" />
> > +  </ItemGroup>
> > +  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
> > +  <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
>
> remove the redundant comment
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/1104/files#r51108048>
> .
>